src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -639,7 +636,10 @@ private void LogRenameSession(RenameLogMessage.UserActionOutcome outcome, bool p
         public void Cancel()
         {
             _threadingContext.ThrowIfNotOnUIThread();
-            VerifyNotDismissed();
+            if (_dismissed)
+            {
+                return;
+            }
```

Comment:
ni: still small preference that this go into DismissUIAndRollbackEditsAndEndRenameSessionAsync, right before DismissUIAndRollbackEdits.  And that method should remove `_dismissed = true;` and move directly into a check in DismissUIAndRollbackEditsAndEndRenameSessionAsync.  this way the checking/setting of this field is isolated to a single place.  Thanks!

Reply:
You got it!